src/linux/process.rs

```diff
@@ -17,6 +17,8 @@ use crate::sys::utils::{
 use crate::utils::into_iter;
 use crate::{DiskUsage, Gid, Pid, ProcessExt, ProcessRefreshKind, ProcessStatus, Signal, Uid};
 
+const HZ: f32 = 100.;
```

Comment:
Not sure to understand the point of this constant. What additional value does it bring?

Reply:
The value goes by the same name in the Linux kernel as the scaling factor used for the external value of the "tick" that the system times expose. The same special value was used in more than one place, so I made it a named constant to indicate what the scale was doing. I can add a comment to the constant if that would help.

Reply:
Oh 🤦🏻 I totally misread that as a divide instead of multiply. Then yes, it very much does not make sense. I'll fix.

Reply:
Thanks!